src/components/containers/Settings.js

```diff
@@ -52,42 +55,57 @@ const DropdownWrapper = styled.div`
 
   & img:hover {
     background-color: #eee;
-  } 
+  }
 `;
 
 const Dropdown = ({title, imageIcon}) => {
   const [hidden, setHidden] = useState(true);
 
   return (
     <DropdownWrapper hide={hidden}>
-      
+
       <div style={{display:'flex' }}>
       <img style={{width:'20px', padding: '5px'}} src={imageIcon}/>
       {title}
       <img style={{width:'20px', padding: '2px', marginLeft: 'auto'}} src="arrow-down.svg" onClick={()=>setHidden(!hidden)}/>
-      
+
       </div>
       <div className="content">
         <br></br>
-        Settings<br></br> 
+        Settings<br></br>
         Content
       </div>
     </DropdownWrapper>
   )
 }
 
 const Settings = () => {
+  /* Sample UserService use: */
+  // UserService.setHours(9, 20, function(res) {
+  //   console.log('Hours set');
+  //   console.log(res);
+  // });
+  //
+  // UserService.setPrimaryCalendar("cal6", function(res) {
+  //   console.log('Primary calendar set');
+  //   console.log(res);
+  // });
+  // UserService.setRelevantCalendars("cal1//cal3//cal6//cal7", function(res) {
```

Comment:
Can you make these comma separated instead of separated by `//` ? Thanks!

Reply:
This is because we need them to be comma separated on the backend and I want the example to reflect correct usage.